includes/class-admin.php

```diff
@@ -9,17 +9,128 @@
+		add_action( 'show_user_profile', array( $this, 'manage_site_options' ), 1 );
+		add_action( 'personal_options_update', array( $this, 'manage_site_options_update' ) );
+		add_action( 'edit_user_profile_update', array( $this, 'manage_site_options_update' ) );
+
+		// add js file
+		add_action( 'admin_enqueue_scripts', array( $this, 'table_enqueue_admin_script' ) );
+	}
+
+	public function table_enqueue_admin_script( $hook ) {
+		wp_enqueue_script( 'script_table', plugin_dir_url( __FILE__ ) . 'js/table.js', array(), '1.0' );
+	}
+
+	/**
+	 * @return void
+	 */
+	public function manage_site_options( $user ) {
+		if ( ! user_can( $user->ID, 'manage_network_users' ) ) {
+			return;
+		}
+		$all_sites         = get_sites();
+		$current_user_site = wp_sub_get_user_sites( $user->ID );
+		$data_site_format  = array();
+		?>
+		<h2><?php _e( 'Manage Sites', 'wp-separate-user-base' ); ?></h2>
+		<div class="wrap">
+			<div class="tablenav top">
+				<div class="alignleft actions bulkactions">
+					<select name="add_site" id="action-selector-site" style="min-width: 200px;">
+						<?php if ( count( $all_sites ) > 0 ) : ?>
+							<?php foreach ( $all_sites as $site ) : ?>
+								<?php $data_site_format[ $site->blog_id ] = $site; ?>
+								<?php if ( ! in_array( $site->blog_id, $current_user_site, true ) ) : ?>
+									<option value="<?php echo esc_html( $site->blog_id ); ?>">
+										<?php echo esc_html( $site->blogname ); ?> <b>(ID: <?php echo esc_html( $site->blog_id ); ?>)</b>
+									</option>
+								<?php endif; ?>
+							<?php endforeach; ?>
+						<?php else : ?>
+							<option ><?php _e( 'Empty site', 'wp-separate-user-base' ); ?></option>
+						<?php endif; ?>
+					</select>
+					<button type="button" id="add_site" class="button action"><?php _e( 'Add', 'wp-separate-user-base' ); ?></button>
+				</div>
+			</div>
+			<table id="list_site" class="wp-list-table widefat fixed striped table-view-list posts" style="width:500px;">
+				<tbody id="the-list">
+				<?php if ( count( $current_user_site ) > 0 ) : ?>
+					<?php foreach ( $current_user_site as $site_id ) : ?>
+						<tr id="site-<?php echo $site_id; ?>" >
+							<td class="title column-title has-row-actions column-primary page-title" data-colname="Site">
+								<?php if ( isset( $data_site_format[ $site_id ] ) ) : ?>
+									<?php echo esc_html( $data_site_format[ $site_id ]->blogname ); ?> (ID: <?php echo esc_html( $data_site_format[ $site_id ]->blog_id ); ?>)
+								<?php else : ?>
+									Site <?php echo esc_html( $site_id ); ?>
+								<?php endif; ?>
+							</td>
+							<td class="action-remove-site" style="width: 80px;" >
+								<input class="input-text" type="hidden" value="<?php echo esc_html( $site_id ); ?>" name="site_id[<?php echo esc_html( $site_id ); ?>]" />
+								<button type="button" class="button action">
+									<span class="btn-remove-site"><?php _e( 'Remove', 'wp-separate-user-base' ); ?></span>
```

Comment:
This also needs to be escaped. There are escaping translation functions like `esc_html_e` for that.